src/psyclone/tests/domain/lfric/kern_call_arg_list_test.py

```diff
@@ -106,6 +106,10 @@ def test_cellmap_intergrid(dist_mem, fortran_writer):
         'map_w2(:,cell)']
 
     check_psyir_results(create_arg_list, fortran_writer)
+    # pylint: disable=no-member
```

Comment:
Again, you can bring this to the top since it happens multiple times in the file, and add a comment that this is because psyir classes are generated at runtime.

Reply:
Done.